pkgs/misc/emulators/mame/default.nix

```
@@ -26,7 +26,7 @@ in mkDerivation {
     owner = "mamedev";
     repo = "mame";
     rev = "mame${majorVersion}${minorVersion}";
-    sha256 = "1pqwa5qcrk0wgbyns6fkv7j3pn7byf216cixc3phhhgvfidgr8ww";
+    sha256 = "1jwc50hmlrkdj204gqcvf7lvgb1r6rhj8hhckykylb1mli04c7gy";
```

Both ofborg and my local machine get a different hash, namely:

```suggestion
    sha256 = "1ij08h7cflr76qzyhhj21948275lqkpzi9r4pbc7h2avrlpsijx4";
```

---

@danieldk Applied, thanks!